### PR TITLE
Maven project version upgrade from 3.0.1-SNAPSHOT to 3.0.2-SNAPSHOT

### PR DESCRIPTION
**Pull request by rfelcman**
plus plugin and dependencies upgrade

### CODE REVIEW

File: oracleddlparser/pom.xml [hunk 1]
```diff
@@ -160,12 +160,12 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-surefire-plugin</artifactId>
-                    <version>3.0.0-M5</version>
+                    <version>3.0.0-M8</version>
```
Comment (lukasj) [inline oracleddlparser/pom.xml:163]: 3.0.0 is out

File: oracleddlparser/pom.xml [hunk 2]
```diff
@@ -175,12 +175,12 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-javadoc-plugin</artifactId>
-                    <version>3.2.0</version>
+                    <version>3.4.1</version>
```
Comment (lukasj) [inline oracleddlparser/pom.xml:178]: 3.5.0 is there I believe

**Review by lukasj — COMMENTED** [review]
one can use `mvn -U -C versions:display-plugin-updates -Poss-release,staging` to get latest versions

**Comment by rfelcman** [comment]
Sorry for this I used `mvn versions:display-plugin-updates` and this one didn't discovered everything.

**Review by lukasj — APPROVED** [review]
LGTM

**Pull request merged**